src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A deadline task that extends from Task.
+ *
+ * @author Erwin Quek
+ * @version CS2103 AY21/22 Sem 1
+ */
+public class Deadline extends Task{
+    private String dateTime;
+    private final LocalDateTime dt;
+
+    /**
+     * Constructor for Deadline.
+     * @param description Takes in a description of Deadline task
+     */
+    public Deadline(String description) {
+        super(description.substring(0, description.indexOf("/by ")).trim());
+        this.dateTime = description.substring(description.indexOf("/by ") + 4).trim();
+        System.out.println(description);
+        this.dt = LocalDateTime.parse(this.dateTime);  //eg LocalDateTime.parse("2015-02-20T06:30");
+    }
+
+    /**
+     * A method to get deadline task info.
```

Comment:
Use "Returns" instead

Reply:
Thank you for that feedback 😄 ! i will update eit